webrecorder/browsertrix
# feat: Allow org admins to set default workflow configs

Resolves https://github.com/webrecorder/browsertrix/issues/2017, follows https://github.com/webrecorder/browsertrix/pull/2031

### Changes

- Adds new org settings tab for updating crawl details
- Refactors `workflow-editor` to move out utility functions
- Updates user guide on org settings

### Manual testing

1. Log in as org admin
2. Go to Settings -> Crawl Workflows
3. Update settings and save
4. Go to New Workflow
5. Click through sections. Verify default values are set as expected
6. Save new workflow. Verify values save as expected
7. Go to Settings -> Crawl Workflows
8. Remove settings and save
9. Go to New Workflow. Verify default values are no longer set
